reward/staking_manager.go

```diff
@@ -108,3 +108,12 @@ func (sm *stakingManager) handleChainHeadEvent() {
 		}
 	}
 }
+
+// Unsubscribe can unsubscribe a subscription to listen chain head event.
+func (sm *StakingManager) Unsubscribe() {
+	sm.chainHeadSub.Unsubscribe()
+}
+
+func (sm *StakingManager) IsSubscribing() bool {
```

Comment:
Method description comment is missing.

Reply:
I'll make a new pr for method description comments